### DISCUSSION TITLE
MIT compatibility with AGPL.

### DISCUSSION THREAD

**Discussion by ghost** [framesurge/perseus#215] [General]
Perseus is released under MIT license. The command "perseus new my-app" generates some source code, if I include the source code generated by "perseus new my-app" into my project, can I release my project under AGPL? Please notice there's a letter "A": It's Affero GPL, which has stronger copyleft than GPL.

**Comment by erlend-sh** [comment]
Yes, that’s fine. It’s the other way around (including AGPL code in MIT-type code) that’s tricky.
